<?xml version="1.0"?>
<launch>
  <node name="ft_sensor" pkg="robotiq_ft_sensor" type="ft_sensor" respawn="true" output="screen">
			<param name="frame_id" type="str" value="robotiq_ft_frame_id" />
			<param name="serial_id" type="str" value="ttyUSB0" />
  </node>


</launch>
